drivers/serial/Kconfig.it8xxx2

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig UART_ITE_IT8XXX2
+	bool "ITE NS16550 serial driver"
+	select SERIAL_HAS_DRIVER
+	select SERIAL_SUPPORT_INTERRUPT
+	help
+	  This option enables the ITE NS16550 serial driver.
+	  This driver can be used for the serial hardware
+	  available on ITE EV-boards.
+
+# ---------- Port 0 ----------
+
+menuconfig UART_ITE_IT8XXX2_PORT_0
+	bool "Enable ITE NS16550 Port 0"
+	depends on UART_ITE_IT8XXX2
+	help
+	  This tells the driver to configure the UART port at boot, depending on
+	  the additional configure options below.
+
+config UART_ITE_IT8XXX2_PORT_0_OPTIONS
+	int "Port 0 Options"
```

Comment:
What is the key for this options bit field. We should point to documenation

Reply:
It has been removed

Reply:
We remove it because it is dummy code on this project.
we use this to verify which port it is on other project.
But in zephyr, the port is treated as different device.